Stepic.xcodeproj/project.pbxproj

```diff
@@ -9507,7 +9507,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 261;
+				CURRENT_PROJECT_VERSION = 262;
```

Comment:
In App Store Connect current `Bundle Version String` = `263`, here should be at minimum `264`.